src/cloud_provider/digitalocean/network/vpc.rs

```diff
@@ -248,11 +248,11 @@ mod tests_do_vpcs {
     #[test]
     fn check_reserved_subnets() {
         // if not reserved
-        assert_eq!(is_do_reserved_vpc_subnets(Region::Frankfurt, "192.168.0.0/24"), false);
+        assert_eq!(is_do_reserved_vpc_subnets(DoRegion::Frankfurt, "192.168.0.0/24"), false);
```

Comment:
Can be `assert_eq!(!is_do_reserved_vpc_subnets(DoRegion::Frankfurt, "192.168.0.0/24"));`